partial/libuptiny/json_common.c

```diff
@@ -0,0 +1,28 @@
+#include "jsmn/jsmn.h"
+
+extern jsmntok_t token_pool[];
+extern const unsigned int token_pool_size;
+
+unsigned int consume_recursive_json(unsigned int idx) {
```

Comment:
Set a maximum recursion depth here?

Reply:
Yes, true.

Reply:
OK, so I've just removed the recursion, it's better than processing another error case.